.github/workflows/android_shared_lint_and_test.yml

```
@@ -32,11 +32,18 @@ jobs:
           restore-keys: |
             ${{ runner.os }}-gradle-
 
-      - name: Setup BuildKonfig Properties
+      - name: Setup Konfig Properties
         env:
-          KMM_KONFIG_PROPERTIES: ${{ secrets.KMM_KONFIG_PROPERTIES }}
+          BUILD_KONFIG: ${{ secrets.BUILD_KONFIG }}
         run: |
-          echo $KMM_KONFIG_PROPERTIES | base64 --decode > buildKonfig.properties
+          echo "$BUILD_KONFIG" > buildSrc/src/main/kotlin/appPackage/BuildKonfig.kt
+
+      - name: Setup Google Services
+        env:
+          GOOGLE_SERVICES_JSON_STAGING: ${{ secrets.GOOGLE_SERVICES_JSON_STAGING }}
+        run: |
+          mkdir -p android/src/staging
+          echo $GOOGLE_SERVICES_JSON_STAGING > android/src/staging/google-services.json
```

Please check my suggestion to see if it works (to make it consistent with build konfig)
```suggestion
          echo "$GOOGLE_SERVICES_JSON_STAGING" > android/src/staging/google-services.json
```